mindsdb/integrations/handlers/symbl_handler/__init__.py

```diff
@@ -0,0 +1,33 @@
+from mindsdb.integrations.libs.const import HANDLER_TYPE
+
+from .__about__ import __version__ as version, __description__ as description
+
+try:
+    from .symbl_handler import (
+        SymblHandler as Handler,
+        connection_args_example,
+        connection_args,
+    )
+
+    import_error = None
+except Exception as e:
+    Handler = None
+    import_error = e
+
+title = "Symbl"
+name = "symbl"
+type = HANDLER_TYPE.DATA
+icon_path = "icon.svg"
```

Comment:
Is this the correct logo? It looks like Docker logo

Reply:
Fixed it